### DISCUSSION TITLE
Account restricted for violation of terms of service

### DISCUSSION THREAD

**Discussion by TohaSupchik** [codewars/codewars.com#2785] [Q&A] [answered]
I am a programming teacher, for which I was blocked, solutions are created together with my students

**Comment by DonaldKellett** [comment, marked as answer]
[Qualified](https://qualified.io/) should suit your use case better. It's the commercial version of Codewars.

**Comment by Blind4Basics** [comment]
> I am a programming teacher, [...], solutions are created together with my students

from the 2 parts of this sentence there is at the very least one that is a lie. Otherwise, can you tell me how you write a solution with your students that is grouped with 2+ years old solutions while you subscribed to CW in Jan of this year...?
